onboarding/index.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: page
+title: Onboarding
+subtitle: Welcome to the team!
+show_sidebar: true
+toc: true
+---
+
+## First steps
+
+1. Go to our [Girder data management](https://data.nutritionallungimmunity.org) collection and register.
+1. Request membership in our Nutritional Lung Immunity [Github organization](https://github.com/NutritionalLungImmunity).
```

Comment:
may want to clarify how to do this. over email or through github or in a lab meeting?